cmake/ace_cppstd.cpp

```diff
@@ -0,0 +1,7 @@
+#define ACE_DOESNT_DEFINE_MAIN 1
+#include <ace/Global_Macros.h>
+
+int main()
+{
+  return 0;
+}
```

Comment:
This could be combined with cplusplus.cpp and have different behavior based on a preprocessor macro